pkg/mcs/scheduling/server/testutil.go

```diff
@@ -50,12 +50,14 @@ func NewTestServer(ctx context.Context, re *require.Assertions, cfg *config.Conf
 // GenerateConfig generates a new config with the given options.
 func GenerateConfig(c *config.Config) (*config.Config, error) {
 	arguments := []string{
+		"--name=" + c.Name,
 		"--listen-addr=" + c.ListenAddr,
 		"--advertise-listen-addr=" + c.AdvertiseListenAddr,
 		"--backend-endpoints=" + c.BackendEndpoints,
 	}
 
 	flagSet := pflag.NewFlagSet("test", pflag.ContinueOnError)
+	flagSet.StringP("name", "", "", "human-readable name for this scheduling member")
```

Comment:
Shall we need a default name?

Reply:
default name set by 

https://github.com/tikv/pd/blob/a39300e04db71a6858f29cbcd000fb5e71f5857f/pkg/mcs/tso/server/config.go#L185-L195

Whereas your commented snippet is for testing to avoid using the same name for the same machine for local testing, I used addr here

https://github.com/tikv/pd/blob/51708b50bf7659653f47f93ad8a1fee964774b64/tests/testutil.go#L87-L88